src/docs/product/integrations/Strapi/index.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Strapi
+sidebar_order: 1
+description: "Learn more about Strapi's Sentry plugin and streamline your content API development with Strapi and Sentry.."
+---
+
+Strapi is the leading open-source headless CMS. It’s 100% Javascript, fully customizable and developer-first.
+
+With the Sentry plugin for Strapi, you can easily configure Sentry within your Strapi Application and get all the context you need to understand what events led to errors. Automatically send errors encountered in your application's end API to Sentry and attach useful metadata to Sentry events. 
```

Comment:
This reads as marketing. Instead, focus on the value to the user in setting up the integration with Sentry. We also need to tell people whether to set up the integration once per org or once per project.

Here's one edit (though you'll need to add the set up per org or per project info):
```suggestion
Sentry's Strapi integration provides the context of what events led to errors. Configure Sentry within your Strapi application to automatically send errors encountered in your application's end API to Sentry with metadata attached.

This integration is set up once per....
```